keystone/server.sls

```diff
@@ -103,9 +103,13 @@ keystone_group:
     - service: keystone_service
   {%- endif %}
 
+{% import_json "/etc/keystone/policy.json" as default_policy %}
```

Comment:
Good idea to use file.serialize but this part probably won't work on clean installation where keystone is not yet installed.